law_system/law_vectorstore.py

```
@@ -232,17 +264,31 @@ def load_existing_vectorstore(self) -> bool:
+                logger.info(f"   - 벡터스토어 디렉토리 발견!")
                 self.vectorstore = Chroma(
                     embedding_function=self.embeddings_model,
                     persist_directory=self.persist_directory,
                     collection_name=self.collection_name
                 )
+                
+                # 저장된 문서 수 확인
+                try:
+                    doc_count = self.vectorstore._collection.count()
+                    logger.info(f"   ✅ 벡터스토어 로드 성공! (저장된 문서: {doc_count}개)")
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)